packages/fxa-auth-server/scripts/template-version-bump.js

```diff
@@ -42,11 +36,7 @@ const stagedTemplates = cp
   })
   .map(templatePath => templatePath.split('/')[5])
   .filter(fileName => !IGNORE.has(fileName))
-  .map(fileName =>
-    templates.generateTemplateName(
```

Comment:
`generateTemplateName` is unnecessary now because the templates have the same name as the strings in metrics.